migrations/2020_01_14_130500_change_permission_groups_add_is_hidden.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Flarum\Database\Migration;
+
+return Migration::addColumns('groups', [
+    'is_hidden' => ['boolean', 'default' => 0]
```

Comment:
Do we want to use a hidden_at datetime instead?

Reply:
Personally, the "event" of hiding the group would not be interesting to me. It's not like a discussion where one could expect this flag to change from time to time / where a record is useful. This is a setting that is probably configured once when creating the group and then forgotten.

Reply:
This is similar to the `is_restricted` property on Tags. I also think boolean makes the most sense here.